mri-analysis.md file for addition to bibliography

rohanbanerjee opened the pull request:
This page in the bibliography will contain reference materials regarding MRI analysis.

jcohenadad commented on bibliography/mri-analysis.md:
```diff
@@ -0,0 +1,5 @@
+
+## Fundamentals
+
+* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.drcmr.dk/bloch](http://www.neuroimagingprimers.org) -- physical copy of the book coming up in Lab soon!
```
markdown hyperlink is incorrect-- you can verify by either clicking on 'view' on GH, or building local site (https://github.com/neuropoly/intranet.neuro.polymtl.ca/wiki#build-website-locally)

rohanbanerjee replied:
I have updated the markdown hyperlink.

jcohenadad commented on bibliography/mri-analysis.md:
```diff
@@ -0,0 +1,5 @@
+
+## Fundamentals
+
+* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.neuroimagingprimers.org) -- physical copy of the book coming up in Lab soon!
```
http://www.neuroimagingprimers.org is not a specific URL to the book...

jcohenadad replied:
book ordered-- arrives in 2d-- so you can remove the 'coming up soon'

jcohenadad commented:
you missed https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/87#discussion_r1054814230

rohanbanerjee commented:
Updated. Since the lab has already bought the book, I have added the link to the example boxes of the book mentioned in the book and their official website.

jcohenadad commented on bibliography/mri-analysis.md:
```diff
@@ -2,4 +2,4 @@
 ## Fundamentals
 
-* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.neuroimagingprimers.org) -- physical copy of the book can be found in the lab library.
+* [Introduction to Neuroimaging Analysis by Mark Jenkinson and Michael Chappel](http://www.neuroimagingprimers.org/examples/introduction-primer-example-boxes/) -- physical copy of the book can be found in the lab library.
```
```suggestion
* [Introduction to Neuroimaging Analysis (Mark Jenkinson and Michael Chappel)](http://www.neuroimagingprimers.org/examples/introduction-primer-example-boxes/) -- physical copy of the book can be found in the lab library.
```
